tests/Resources/ti.locale.addontest.js

```diff
@@ -0,0 +1,118 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2015-Present by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
+
+describe('Titanium.Locale', function () {
+	describe('#parseDecimal()', () => {
+		it('compared with String.formatDecimal()', () => {
+			should(Ti.Locale.parseDecimal).be.a.Function();
+
+			const numericValue = 1234567.8;
+
+			let numericString = String.formatDecimal(numericValue);
+			let parsedValue = Ti.Locale.parseDecimal(numericString);
+			should(parsedValue).be.a.Number();
+			should(Math.abs(parsedValue - numericValue)).be.lessThan(Number.EPSILON);
```

Comment:
Neat trick - never knew about the use of EPSILON to compare floats.

May be easier to write this as:
``` js
should(parsedValue).be.approximately(numericValue, Number.EPSILON);
```
See https://shouldjs.github.io/#assertion-approximately